src/main/java/edu/iis/mto/lab_0/TailsGenerator.java

```diff
@@ -6,7 +6,14 @@
 public class TailsGenerator {
 
     public List<String> tails(String value) {
-        return new ArrayList<>(value.length());
+        ArrayList<String> myArray = new ArrayList<>(value.length());
+        myArray.add(value);
+        for (int i = 1; i<value.length();i++)
+        {
+            myArray.add("Ram");

```

Comment:
To jest niepoprawna implementacja funkcji tails. Nieważne jaki parametr zostanie przekazany zachowanie będzie takie samo. Poza tym testy nie przechodzą.